.lacework/codesec.yaml

```diff
@@ -1,4 +1,5 @@
 default:
   sca:
-    enable-dynamic: true
-    enable-fast: true
+    scan:
```

Comment:
Actually, we can probably delete this file instead of changing the format.

Reply:
this file is not being used?

Reply:
They are used for the e2e tests. What happens if you delete this file?

Reply:
I don't know much about the repo, I can delete and try.

Reply:
seems like it is not for the tests but the repo uses it to run those checks in github action.

Reply:
I see that you added it.

https://github.com/lacework/code-security-action/commit/7bf79db5c2970cbf288c77dc9ffd467b33ed6445

Reply:
Yes, I did but back then FAST was not enabled by default. I do not know about dynamic, maybe we need to keep this one. Are the corresponding triggered by these changes?